### ISSUE TITLE
Different views for the post list - images with fixed height

### ISSUE THREAD

**Issue by dotvhs** [Tunous/Dank#25]
I know it might be too early for this, but one of the things I don't like in Dank is that browsing image heavy subreddits is a bit less comfortable than in other clients. So I'd like to suggest adding new view type with bigger images, that have fixed height. Since I can't code, I made some designs for it:

### Image in the middle with rounded corners
![fix height image view 0](https://user-images.githubusercontent.com/681416/51581967-782db800-1eca-11e9-91d2-4b8c50d6fd7f.png)

### Image in the middle, full width
![fix height image view 2](https://user-images.githubusercontent.com/681416/51581997-88459780-1eca-11e9-8dd1-6b01023ef597.png)

### Image at the top, full width
![fix height image view 1](https://user-images.githubusercontent.com/681416/51582016-95fb1d00-1eca-11e9-96c5-e1ceebd338fb.png)

**Comment by Soneliem** [comment]
This is the only reason why I am not using dank. would also be great if gifs and videos could also autoplay in the home screen as well.

**Comment by Tunous** [comment]
First working prototype:

<img src="https://user-images.githubusercontent.com/5156340/62723902-09e9b680-ba12-11e9-8a48-e40d1c727122.png" height="580" />

Entry in preferences:

<img src="https://user-images.githubusercontent.com/5156340/62723903-09e9b680-ba12-11e9-9e9b-4730c702cd26.png" height="580" />

Available options:

<img src="https://user-images.githubusercontent.com/5156340/62723905-09e9b680-ba12-11e9-9c0c-fdd96dc5103f.png" height="580" />
 (have to find good images for these)

**Comment by saket** [comment]
Really nice!

**Comment by Tunous** [comment]
@dotvhs Do you have any ideas how the image style selector presented above could be improved? I'm wondering whether we could make it so it would also display how submissions will look like with each option.

**Comment by dotvhs** [comment]
@Tunous yeah i have an idea. I'm currently on holidays, so I can't work on anything until Monday. But I'll upload some suggestions here next week.
